pmm/pmm2-server-autobuild.groovy

```diff
@@ -118,6 +118,7 @@ pipeline {
                             build-server-rpm pmm-server
                             build-server-rpm pmm-update
                             build-server-rpm dbaas-controller
```

Comment:
Please check that we have that line in other places too. I think we miss it somewhere